deploy/4.dummyPeriphery.ts

```diff
@@ -0,0 +1,54 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
```

Comment:
I think it's better if it is created as needed, do you think it's better to do it via a hardhat task then instead of a deployment script or is there a way to do it via a unit test @CyclopsRex ?

Reply:
Ideal would be a UT for testing migration and the old periphery could be set up in the `before`block off the UT. But if we are just testing migration manually - which is defensible given it is a one-time effort - then this could be created in a task instead of a deploy script.